services/skus/storage/repository/repository.go

```diff
@@ -0,0 +1,252 @@
+// Package repository provides access to data available in SQL-based data store.
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	uuid "github.com/satori/go.uuid"
+	"github.com/shopspring/decimal"
+
+	"github.com/brave-intl/bat-go/libs/datastore"
+
+	"github.com/brave-intl/bat-go/services/skus/model"
+)
+
+type Order struct{}
+
+func NewOrder() *Order { return &Order{} }
```

Comment:
Maybe I am missing something/interpreting this wrong, but where a type only exists to implement an interface i.e. will never have exported methods beyond the interface do we not want to return the interface. For example, this constructor would return OrderStore interface as there is no need to export the type itself. This signals there is nothing of interest beyond the interface.

Reply:
Just a note that we've discussed this, and decided to keep as it is.